test/cases/api.sh

```diff
@@ -152,8 +152,17 @@ function instanceCheck() {
   # Check if postgres is installed
   $_ssh rpm -q postgresql
 
-  # Verify subscribe status
-  subscribe_org_id=$($_ssh sudo subscription-manager identity | grep 'org ID')
+  # Verify subscribe status. Loop check since the system may not be registered such early
+   for LOOP_COUNTER in {1..10}; do
+    subscribe_org_id=$($_ssh sudo subscription-manager identity | grep 'org ID')
+    if [[ "$subscribe_org_id" == "org ID: $API_TEST_SUBSCRIPTION_ORG_ID" ]]; then
+      echo "System is subscribed."
+      break
+    else
+      echo "System is not subscribed. Retrying in 10 seconds...($LOOP_COUNTER/10)"
+      sleep 10
```

Comment:
I prefer a bit larger timeout, see my comment on the issue but otherwise looks good.

Reply:
Let's merge and see. If it still happens we can try increasing